clients/roscpp/src/libros/service_server_link.cpp

```diff
@@ -381,6 +402,11 @@ bool ServiceServerLink::call(const SerializedMessage& req, SerializedMessage& re
   return info->success_;
 }
 
+bool ServiceServerLink::call(const SerializedMessage& req, SerializedMessage& resp)
+{
+  return call(req, resp, -1);
+}
```

Comment:
To keep the order the same as in the header move this function above the new function signature.